protobuf/scalapb/scalapb.proto

```diff
@@ -85,6 +85,10 @@ message ScalaPbOptions {
   // If true, no default values will be generated in message constructors.
   optional bool no_default_values_in_constructor = 15;
 
+  // If false, oneof fields appear last in the constructor parameters list for
+  // generated messages, no matter where they are declared in the proto file.
+  optional bool follow_field_declaration_order = 16 [default=true];
```

Comment:
this is now a bit inconsistent with the freshly-introduced, "negative" `no_default_values_in_constructor`, but follows the style of `lenses` - I have a preference for avoiding negation in the flag name.

Reply:
Yeah, I am not a fan of the negation in the flag name either. However, when the default value is `true`, it gets the `no_` suffix when it becomes a generator parameter which makes it a little inconsistent, so I have preference that the default value is false. Maybe you change it to `oneofs_after_fields_in_constructor`? Then it's non-negated and defaults to false. Open to other naming suggestions that follow the guidelines. I am thinking to rename `no_default_values_in_constructor` to something like `require_values_in_constructor` before the release.